frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.css

```diff
@@ -26,6 +25,20 @@
 	font-size: 1rem;
 }
 
-.level-mission-info-banner:hover {
-	background-color: var(--level-banner-background-colour-hover);
+.level-mission-info-banner .themed-button {
+	height: auto;
+	white-space: nowrap;
```

Comment:
You might need some explicit padding here; I used `0.75rem 1rem` in my trials, which looked ok.